### PR TITLE
systemd doc update

### PR DESCRIPTION
**Pull request by bmillemathias**
##### SUMMARY
Document that systemd module manage units (timers, services, ...) and not only service.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
systemd

### CODE REVIEW

File: lib/ansible/modules/systemd.py [hunk 1]
```diff
@@ -77,20 +78,20 @@
...
     - A system managed by systemd.
 '''
 
 EXAMPLES = '''
-- name: Make sure a service is running
+- name: Make sure a service unit is running
   ansible.builtin.systemd:
     state: started
-    name: httpd
+    name: httpd.service
```
Comment (bcoca) [inline lib/ansible/modules/systemd.py]: I would not add .service to all examples

File: lib/ansible/modules/systemd.py [hunk 2]
```diff
@@ -13,25 +13,26 @@
 author:
     - Ansible Core Team
 version_added: "2.2"
-short_description:  Manage services
+short_description:  Manage systemd units
 description:
-    - Controls systemd services on remote hosts.
+    - Controls systemd units (services, timers, ...) on remote hosts.
```
Comment (Akasurde) [inline lib/ansible/modules/systemd.py]:
```suggestion
    - Controls systemd units (services, timers, and so on) on remote hosts.
```

File: lib/ansible/modules/systemd.py [hunk 3]
```diff
@@ -13,25 +13,26 @@
...
 options:
     name:
         description:
-            - Name of the service. This parameter takes the name of exactly one service to work with.
-            - When using in a chroot environment you always need to specify the full name i.e. (crond.service).
+            - Name of the unit. This parameter takes the name of exactly one unit to work with.
+            - When no extension is given, it is implied to a `.service` as systemd.
```
Comment (Akasurde) [inline lib/ansible/modules/systemd.py]:
```suggestion
            - When no extension is given, it is implied to a C(.service) as systemd.
```

File: lib/ansible/modules/systemd.py [hunk 4]
```diff
@@ -13,25 +13,26 @@
...
+            - When using in a chroot environment you always need to specify the name of the unit with the extension (``crond.service``).
```
Comment (Akasurde) [inline lib/ansible/modules/systemd.py]:
```suggestion
            - When using in a chroot environment you always need to specify the name of the unit with the extension. For example, C(crond.service).
```

File: lib/ansible/modules/systemd.py [hunk 5]
```diff
@@ -77,17 +78,17 @@
         default: no
         version_added: "2.3"
 notes:
-    - Since 2.4, one of the following options is required 'state', 'enabled', 'masked', 'daemon_reload', ('daemon_reexec' since 2.8),
-      and all except 'daemon_reload' (and 'daemon_reexec' since 2.8) also require 'name'.
-    - Before 2.4 you always required 'name'.
+    - Since 2.4, one of the following options is required C(state), C(enabled), C(masked), C(daemon_reload), (C(daemon_reexec) since 2.8),
+      and all except C(daemon_reload) (and 'daemon_reexec' since 2.8) also require C(name).
```
Comment (Akasurde) [inline lib/ansible/modules/systemd.py]:
```suggestion
      and all except C(daemon_reload) and (C(daemon_reexec) since 2.8) also require C(name).
```

**Review by Akasurde — DISMISSED** [review]
Rest LGTM

File: lib/ansible/modules/systemd.py (hunk 2 above)
Comment (bcoca) [reply lib/ansible/modules/systemd.py]: ...  is not universally understood, but 'and so on' can be translated

**Comment by Akasurde** [comment]
@bmillemathias Thanks for the contribution. @acozine @bcoca Thanks for the review.

**Pull request merged**